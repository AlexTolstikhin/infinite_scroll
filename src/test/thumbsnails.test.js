import React from 'react';
import { shallow } from 'enzyme';
import Thumbsnails from '../components/thumbsnails';

describe('Thumbsnails', () => {
	it('should match snapshot', () => {
		const props = {
			listHeight: 600,
			listWidth: 900,
			numberOfRowsToRender: 10,
			rowHeight: 350,
			thumbsInRow: 10
		}
		const wrapper = shallow(<Thumbsnails {...props}/>)
		expect(wrapper.debug()).toMatchSnapshot();
	});
});