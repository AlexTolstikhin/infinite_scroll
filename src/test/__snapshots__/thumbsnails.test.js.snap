// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thumbsnails should match snapshot 1`] = `
"<div>
  <div onScroll={[Function: debounced]} style={{...}}>
    <div style={{...}}>
      <div style={{...}}>
        <img alt={1500348280} src=\\"http://hiring.verkada.com/thumbs/1500348280.jpg\\" style={{...}} />
        <img alt={1500348300} src=\\"http://hiring.verkada.com/thumbs/1500348300.jpg\\" style={{...}} />
        <img alt={1500348320} src=\\"http://hiring.verkada.com/thumbs/1500348320.jpg\\" style={{...}} />
        <img alt={1500348340} src=\\"http://hiring.verkada.com/thumbs/1500348340.jpg\\" style={{...}} />
        <img alt={1500348360} src=\\"http://hiring.verkada.com/thumbs/1500348360.jpg\\" style={{...}} />
        <img alt={1500348380} src=\\"http://hiring.verkada.com/thumbs/1500348380.jpg\\" style={{...}} />
        <img alt={1500348400} src=\\"http://hiring.verkada.com/thumbs/1500348400.jpg\\" style={{...}} />
        <img alt={1500348420} src=\\"http://hiring.verkada.com/thumbs/1500348420.jpg\\" style={{...}} />
        <img alt={1500348440} src=\\"http://hiring.verkada.com/thumbs/1500348440.jpg\\" style={{...}} />
        <img alt={1500348460} src=\\"http://hiring.verkada.com/thumbs/1500348460.jpg\\" style={{...}} />
      </div>
      <div style={{...}}>
        <img alt={1500348480} src=\\"http://hiring.verkada.com/thumbs/1500348480.jpg\\" style={{...}} />
        <img alt={1500348500} src=\\"http://hiring.verkada.com/thumbs/1500348500.jpg\\" style={{...}} />
        <img alt={1500348520} src=\\"http://hiring.verkada.com/thumbs/1500348520.jpg\\" style={{...}} />
        <img alt={1500348540} src=\\"http://hiring.verkada.com/thumbs/1500348540.jpg\\" style={{...}} />
        <img alt={1500348560} src=\\"http://hiring.verkada.com/thumbs/1500348560.jpg\\" style={{...}} />
        <img alt={1500348580} src=\\"http://hiring.verkada.com/thumbs/1500348580.jpg\\" style={{...}} />
        <img alt={1500348600} src=\\"http://hiring.verkada.com/thumbs/1500348600.jpg\\" style={{...}} />
        <img alt={1500348620} src=\\"http://hiring.verkada.com/thumbs/1500348620.jpg\\" style={{...}} />
        <img alt={1500348640} src=\\"http://hiring.verkada.com/thumbs/1500348640.jpg\\" style={{...}} />
        <img alt={1500348660} src=\\"http://hiring.verkada.com/thumbs/1500348660.jpg\\" style={{...}} />
      </div>
      <div style={{...}}>
        <img alt={1500348680} src=\\"http://hiring.verkada.com/thumbs/1500348680.jpg\\" style={{...}} />
        <img alt={1500348700} src=\\"http://hiring.verkada.com/thumbs/1500348700.jpg\\" style={{...}} />
        <img alt={1500348720} src=\\"http://hiring.verkada.com/thumbs/1500348720.jpg\\" style={{...}} />
        <img alt={1500348740} src=\\"http://hiring.verkada.com/thumbs/1500348740.jpg\\" style={{...}} />
        <img alt={1500348760} src=\\"http://hiring.verkada.com/thumbs/1500348760.jpg\\" style={{...}} />
        <img alt={1500348780} src=\\"http://hiring.verkada.com/thumbs/1500348780.jpg\\" style={{...}} />
        <img alt={1500348800} src=\\"http://hiring.verkada.com/thumbs/1500348800.jpg\\" style={{...}} />
        <img alt={1500348820} src=\\"http://hiring.verkada.com/thumbs/1500348820.jpg\\" style={{...}} />
        <img alt={1500348840} src=\\"http://hiring.verkada.com/thumbs/1500348840.jpg\\" style={{...}} />
        <img alt={1500348860} src=\\"http://hiring.verkada.com/thumbs/1500348860.jpg\\" style={{...}} />
      </div>
      <div style={{...}}>
        <img alt={1500348880} src=\\"http://hiring.verkada.com/thumbs/1500348880.jpg\\" style={{...}} />
        <img alt={1500348900} src=\\"http://hiring.verkada.com/thumbs/1500348900.jpg\\" style={{...}} />
        <img alt={1500348920} src=\\"http://hiring.verkada.com/thumbs/1500348920.jpg\\" style={{...}} />
        <img alt={1500348940} src=\\"http://hiring.verkada.com/thumbs/1500348940.jpg\\" style={{...}} />
        <img alt={1500348960} src=\\"http://hiring.verkada.com/thumbs/1500348960.jpg\\" style={{...}} />
        <img alt={1500348980} src=\\"http://hiring.verkada.com/thumbs/1500348980.jpg\\" style={{...}} />
        <img alt={1500349000} src=\\"http://hiring.verkada.com/thumbs/1500349000.jpg\\" style={{...}} />
        <img alt={1500349020} src=\\"http://hiring.verkada.com/thumbs/1500349020.jpg\\" style={{...}} />
        <img alt={1500349040} src=\\"http://hiring.verkada.com/thumbs/1500349040.jpg\\" style={{...}} />
        <img alt={1500349060} src=\\"http://hiring.verkada.com/thumbs/1500349060.jpg\\" style={{...}} />
      </div>
      <div style={{...}}>
        <img alt={1500349080} src=\\"http://hiring.verkada.com/thumbs/1500349080.jpg\\" style={{...}} />
        <img alt={1500349100} src=\\"http://hiring.verkada.com/thumbs/1500349100.jpg\\" style={{...}} />
        <img alt={1500349120} src=\\"http://hiring.verkada.com/thumbs/1500349120.jpg\\" style={{...}} />
        <img alt={1500349140} src=\\"http://hiring.verkada.com/thumbs/1500349140.jpg\\" style={{...}} />
        <img alt={1500349160} src=\\"http://hiring.verkada.com/thumbs/1500349160.jpg\\" style={{...}} />
        <img alt={1500349180} src=\\"http://hiring.verkada.com/thumbs/1500349180.jpg\\" style={{...}} />
        <img alt={1500349200} src=\\"http://hiring.verkada.com/thumbs/1500349200.jpg\\" style={{...}} />
        <img alt={1500349220} src=\\"http://hiring.verkada.com/thumbs/1500349220.jpg\\" style={{...}} />
        <img alt={1500349240} src=\\"http://hiring.verkada.com/thumbs/1500349240.jpg\\" style={{...}} />
        <img alt={1500349260} src=\\"http://hiring.verkada.com/thumbs/1500349260.jpg\\" style={{...}} />
      </div>
      <div style={{...}}>
        <img alt={1500349280} src=\\"http://hiring.verkada.com/thumbs/1500349280.jpg\\" style={{...}} />
        <img alt={1500349300} src=\\"http://hiring.verkada.com/thumbs/1500349300.jpg\\" style={{...}} />
        <img alt={1500349320} src=\\"http://hiring.verkada.com/thumbs/1500349320.jpg\\" style={{...}} />
        <img alt={1500349340} src=\\"http://hiring.verkada.com/thumbs/1500349340.jpg\\" style={{...}} />
        <img alt={1500349360} src=\\"http://hiring.verkada.com/thumbs/1500349360.jpg\\" style={{...}} />
        <img alt={1500349380} src=\\"http://hiring.verkada.com/thumbs/1500349380.jpg\\" style={{...}} />
        <img alt={1500349400} src=\\"http://hiring.verkada.com/thumbs/1500349400.jpg\\" style={{...}} />
        <img alt={1500349420} src=\\"http://hiring.verkada.com/thumbs/1500349420.jpg\\" style={{...}} />
        <img alt={1500349440} src=\\"http://hiring.verkada.com/thumbs/1500349440.jpg\\" style={{...}} />
        <img alt={1500349460} src=\\"http://hiring.verkada.com/thumbs/1500349460.jpg\\" style={{...}} />
      </div>
      <div style={{...}}>
        <img alt={1500349480} src=\\"http://hiring.verkada.com/thumbs/1500349480.jpg\\" style={{...}} />
        <img alt={1500349500} src=\\"http://hiring.verkada.com/thumbs/1500349500.jpg\\" style={{...}} />
        <img alt={1500349520} src=\\"http://hiring.verkada.com/thumbs/1500349520.jpg\\" style={{...}} />
        <img alt={1500349540} src=\\"http://hiring.verkada.com/thumbs/1500349540.jpg\\" style={{...}} />
        <img alt={1500349560} src=\\"http://hiring.verkada.com/thumbs/1500349560.jpg\\" style={{...}} />
        <img alt={1500349580} src=\\"http://hiring.verkada.com/thumbs/1500349580.jpg\\" style={{...}} />
        <img alt={1500349600} src=\\"http://hiring.verkada.com/thumbs/1500349600.jpg\\" style={{...}} />
        <img alt={1500349620} src=\\"http://hiring.verkada.com/thumbs/1500349620.jpg\\" style={{...}} />
        <img alt={1500349640} src=\\"http://hiring.verkada.com/thumbs/1500349640.jpg\\" style={{...}} />
        <img alt={1500349660} src=\\"http://hiring.verkada.com/thumbs/1500349660.jpg\\" style={{...}} />
      </div>
      <div style={{...}}>
        <img alt={1500349680} src=\\"http://hiring.verkada.com/thumbs/1500349680.jpg\\" style={{...}} />
        <img alt={1500349700} src=\\"http://hiring.verkada.com/thumbs/1500349700.jpg\\" style={{...}} />
        <img alt={1500349720} src=\\"http://hiring.verkada.com/thumbs/1500349720.jpg\\" style={{...}} />
        <img alt={1500349740} src=\\"http://hiring.verkada.com/thumbs/1500349740.jpg\\" style={{...}} />
        <img alt={1500349760} src=\\"http://hiring.verkada.com/thumbs/1500349760.jpg\\" style={{...}} />
        <img alt={1500349780} src=\\"http://hiring.verkada.com/thumbs/1500349780.jpg\\" style={{...}} />
        <img alt={1500349800} src=\\"http://hiring.verkada.com/thumbs/1500349800.jpg\\" style={{...}} />
        <img alt={1500349820} src=\\"http://hiring.verkada.com/thumbs/1500349820.jpg\\" style={{...}} />
        <img alt={1500349840} src=\\"http://hiring.verkada.com/thumbs/1500349840.jpg\\" style={{...}} />
        <img alt={1500349860} src=\\"http://hiring.verkada.com/thumbs/1500349860.jpg\\" style={{...}} />
      </div>
      <div style={{...}}>
        <img alt={1500349880} src=\\"http://hiring.verkada.com/thumbs/1500349880.jpg\\" style={{...}} />
        <img alt={1500349900} src=\\"http://hiring.verkada.com/thumbs/1500349900.jpg\\" style={{...}} />
        <img alt={1500349920} src=\\"http://hiring.verkada.com/thumbs/1500349920.jpg\\" style={{...}} />
        <img alt={1500349940} src=\\"http://hiring.verkada.com/thumbs/1500349940.jpg\\" style={{...}} />
        <img alt={1500349960} src=\\"http://hiring.verkada.com/thumbs/1500349960.jpg\\" style={{...}} />
        <img alt={1500349980} src=\\"http://hiring.verkada.com/thumbs/1500349980.jpg\\" style={{...}} />
        <img alt={1500350000} src=\\"http://hiring.verkada.com/thumbs/1500350000.jpg\\" style={{...}} />
        <img alt={1500350020} src=\\"http://hiring.verkada.com/thumbs/1500350020.jpg\\" style={{...}} />
        <img alt={1500350040} src=\\"http://hiring.verkada.com/thumbs/1500350040.jpg\\" style={{...}} />
        <img alt={1500350060} src=\\"http://hiring.verkada.com/thumbs/1500350060.jpg\\" style={{...}} />
      </div>
      <div style={{...}}>
        <img alt={1500350080} src=\\"http://hiring.verkada.com/thumbs/1500350080.jpg\\" style={{...}} />
        <img alt={1500350100} src=\\"http://hiring.verkada.com/thumbs/1500350100.jpg\\" style={{...}} />
        <img alt={1500350120} src=\\"http://hiring.verkada.com/thumbs/1500350120.jpg\\" style={{...}} />
        <img alt={1500350140} src=\\"http://hiring.verkada.com/thumbs/1500350140.jpg\\" style={{...}} />
        <img alt={1500350160} src=\\"http://hiring.verkada.com/thumbs/1500350160.jpg\\" style={{...}} />
        <img alt={1500350180} src=\\"http://hiring.verkada.com/thumbs/1500350180.jpg\\" style={{...}} />
        <img alt={1500350200} src=\\"http://hiring.verkada.com/thumbs/1500350200.jpg\\" style={{...}} />
        <img alt={1500350220} src=\\"http://hiring.verkada.com/thumbs/1500350220.jpg\\" style={{...}} />
        <img alt={1500350240} src=\\"http://hiring.verkada.com/thumbs/1500350240.jpg\\" style={{...}} />
        <img alt={1500350260} src=\\"http://hiring.verkada.com/thumbs/1500350260.jpg\\" style={{...}} />
      </div>
    </div>
  </div>
</div>"
`;
